tests/_support/AcceptanceTester.php

```diff
@@ -18,9 +18,39 @@
 */
 class AcceptanceTester extends \Codeception\Actor
 {
-    use _generated\AcceptanceTesterActions;
+	use _generated\AcceptanceTesterActions;
 
-    /**
-     * Define custom actions here
-     */
+	public function fillTinyMceEditorById($id, $content) {
+		$this->fillTinyMceEditor('id', $id, $content);
+	}
+
+	public function fillTinyMceEditorByName($name, $content) {
+		$this->fillTinyMceEditor('name', $name, $content);
+	}
+
+	private function fillTinyMceEditor($attribute, $value, $content) {
+		$this->fillRteEditor(
+			\Facebook\WebDriver\WebDriverBy::xpath(
+				'//textarea[@' . $attribute . '=\'' . $value . '\']/../div[contains(@class, \'mce-tinymce\')]//iframe'
+			),
+			$content
+		);
+	}
+
+	private function fillRteEditor($selector, $content) {
+		$this->executeInSelenium(
+			function (\Facebook\WebDriver\Remote\RemoteWebDriver $webDriver)
+			use ($selector, $content) {
+				$webDriver->switchTo()->frame(
+					$webDriver->findElement($selector)
+				);
+
+				$webDriver->executeScript(
+					'arguments[0].innerHTML = "' . addslashes($content) . '"',
+					[$webDriver->findElement(\Facebook\WebDriver\WebDriverBy::tagName('body'))]
+				);
+
+				$webDriver->switchTo()->defaultContent();
+			});
+	}
```

Comment:
This is a first draft on how to fill TinyMce editors. It should work once we find a way to enable TinyMce on Codeception tests.